playwright/models/FirmwareUpdateModal.ts

```diff
@@ -0,0 +1,23 @@
+import { Page, Locator } from "@playwright/test";
+import { Modal } from "./Modal";
+
+export class FirmwareUpdateModal extends Modal {
+  readonly page: Page;
+  readonly readyCheckbox: Locator;
+  readonly downloadProgress: Locator;
+  readonly flashProgress: Locator;
+  readonly updateDone: Locator;
+
+  constructor(page: Page) {
+    super(page);
+    this.page = page;
+    this.readyCheckbox = page.locator('data-test-id=firmware-update-ready-checkbox');
+    this.downloadProgress = page.locator('data-test-id=firmware-update-download-progress');
```

Comment:
Any need for these three?

Reply:
They're used by po itself and spec file too.